src/components/Card.js

```diff
@@ -17,42 +33,92 @@ class Card {
     this._cardImageEl = this._cardElement.querySelector(".card__image-card");
     this._cardImageEl.src = this._link;
     this._cardImageEl.alt = this._title;
+    this._cardLikesEl = this._cardElement.querySelector(".card__button-count");
+    this._cardLikesEl.textContent = this._getLikes();
+    this._trashEl = this._cardElement.querySelector(".card__button-del");
+    if (this._cardOwnerId !== this._myId) {
+      this._trashEl.classList.add("card__button-del-inactive");
+    }
+
+    if (this.isLiked()) {
+      this._cardElement
+        .querySelector(".card__button-like")
+        .classList.add("card__button-like_active");
+    }
 
     this._setEventListeners();
 
     return this._cardElement;
   }
 
   _setEventListeners() {
-    this._cardElement
+    this._cardElement // handle a click on the like button
       .querySelector(".card__button-like")
       .addEventListener("click", () => {
-        this._handleLikeIcon();
+        this._handleLike(); // Line 76
       });
 
-    this._cardElement
+    this._cardElement // handle a click on the trashcan
       .querySelector(".card__button-del")
       .addEventListener("click", () => {
-        this._handleDeleteCard();
+        this._handleDeleteClick(this, this._id);
       });
 
-    this._cardElement
+    this._cardElement // handle clicking for a large image
       .querySelector(".card__image-display")
       .addEventListener("click", () =>
         this._handleImageClick(this._title, this._link)
       );
   }
 
-  _handleLikeIcon() {
+  _handleLike() {
     this._cardElement
       .querySelector(".card__button-like")
       .classList.toggle("card__button-like_active");
```

Comment:
You need to **toggle likes** only in block `then` after a successful response from the server because if there is an error the user will not know about that and will see the toggled heart color.

Reply:
Likes now toggles after successful response